templates/blog/author.html

```diff
@@ -9,7 +9,7 @@
   <div class="row">
     <div class="col-8">
       <div class="p-media-object u-no-margin--bottom">
-        <img src="{% if author.id == 217 %}{{ ASSET_SERVER_URL }}60d9b81e-picto-canonical.svg{% else %}{{ author.avatar_urls.96 }}{% endif %}" class="p-media-object__image is-round">
+        <img src="{% if author.id == 217 %}https://assets.ubuntu.com/v1/60d9b81e-picto-canonical.svg{% else %}{{ author.avatar_urls.96 }}{% endif %}" class="p-media-object__image is-round">
```

Comment:
Same comment as on `article.html`: `article.author.avatar_urls.96` doesn't seem to work, though `article.author.avatar_urls['96']` does.